src/rpc/governance.cpp

```diff
@@ -26,358 +26,467 @@
+    throw std::runtime_error(
+                "gobject count (\"mode\")\n"
+                "Count governance objects and votes\n"
+                "\nArguments:\n"
+                "1. \"mode\"   (string, optional, default: \"json\") Output format: json (\"json\") or string in free form (\"all\")\n"
                 );
+}
 
+UniValue gobject_count(const JSONRPCRequest& request)
+{
+    if (request.fHelp)
+        gobject_count_help();
 
-    if(strCommand == "count") {
-        std::string strMode{"json"};
+    std::string strMode{"json"};
 
-        if (request.params.size() == 2) {
-            strMode = request.params[1].get_str();
-        }
+    if (request.params.size() == 2) {
+        strMode = request.params[1].get_str();
+    }
 
-        if (request.params.size() > 2 || (strMode != "json" && strMode != "all")) {
-            throw JSONRPCError(RPC_INVALID_PARAMETER, "Correct usage is 'gobject count ( \"json\"|\"all\" )'");
-        }
+    if (request.params.size() > 2 || (strMode != "json" && strMode != "all"))
+        gobject_count_help();
 
-        return strMode == "json" ? governance.ToJson() : governance.ToString();
-    }
-    /*
-        ------ Example Governance Item ------
+    return strMode == "json" ? governance.ToJson() : governance.ToString();
+}
 
-        gobject submit 6e622bb41bad1fb18e7f23ae96770aeb33129e18bd9efe790522488e580a0a03 0 1 1464292854 "beer-reimbursement" 5b5b22636f6e7472616374222c207b2270726f6a6563745f6e616d65223a20225c22626565722d7265696d62757273656d656e745c22222c20227061796d656e745f61646472657373223a20225c225879324c4b4a4a64655178657948726e34744744514238626a6876464564615576375c22222c2022656e645f64617465223a202231343936333030343030222c20226465736372697074696f6e5f75726c223a20225c227777772e646173687768616c652e6f72672f702f626565722d7265696d62757273656d656e745c22222c2022636f6e74726163745f75726c223a20225c22626565722d7265696d62757273656d656e742e636f6d2f3030312e7064665c22222c20227061796d656e745f616d6f756e74223a20223233342e323334323232222c2022676f7665726e616e63655f6f626a6563745f6964223a2037342c202273746172745f64617465223a202231343833323534303030227d5d5d1
-    */
+void gobject_deserialize_help()
+{
+    throw std::runtime_error(
+                "gobject deserialize \"hex_data\"\n"
+                "Deserialize governance object from hex string to JSON\n"
+                "\nArguments:\n"
+                "1. \"hex_data\"   (string, required) data in hex string form\n"
+                );
+}
 
-    // DEBUG : TEST DESERIALIZATION OF GOVERNANCE META DATA
-    if(strCommand == "deserialize")
-    {
-        if (request.params.size() != 2) {
-            throw JSONRPCError(RPC_INVALID_PARAMETER, "Correct usage is 'gobject deserialize <data-hex>'");
-        }
+UniValue gobject_deserialize(const JSONRPCRequest& request)
+{
+    if (request.fHelp)
+        gobject_deserialize_help();
 
-        std::string strHex = request.params[1].get_str();
+    if (request.params.size() != 2)
```

Comment:
Can squash this `if` and `if (request.fHelp)` above into one `||`. Same for a couple of others below.